packages/UI/src/components/organisms/OpenResponse/StyledOpenResponse.ts

```diff
@@ -0,0 +1,86 @@
+import styled, { css } from 'styled-components';
+import { typographyStyles } from '../../atoms/Typography/Styled.Typography';
+import type { OpenResponseProps } from '.';
+export const Label = styled.label`
+  ${typographyStyles['label']};
+  margin-left: 1rem;
```

Comment:
@Nouri-Anouar why are we doing margin left of 1 rem?

Reply:
I solved that issue